nncf/openvino/graph/layer_attributes.py

```diff
@@ -79,51 +81,71 @@ def get_weighted_layer_attributes(
     :param constant_attributes: Constant attributes collected for the given node.
     :return: Weighted layer attributes for the given node.
     """
-    retval = {}
-    for port_id, attrs in constant_attributes.items():
-        if ov_metatype in [
-            OVConvolutionMetatype,
-            OVDepthwiseConvolutionMetatype,
-            OVGroupConvolutionMetatype,
-            OVConvolutionBackpropDataMetatype,
-            OVGroupConvolutionBackpropDataMetatype,
-        ]:
-            node_attrs = ov_node.get_attributes()
-            kwargs = {
-                "weight_requires_grad": False,
-                "stride": tuple(node_attrs["strides"]),
-                "dilations": node_attrs["dilations"],
-                "transpose": ov_metatype in [OVConvolutionBackpropDataMetatype, OVGroupConvolutionBackpropDataMetatype],
-                # TODO: ticket 114378: unify pad attribute
-                "padding_values": tuple(node_attrs["pads_begin"] + node_attrs["pads_end"]),
+    if len(constant_attributes) != 1:
+        return None
+
+    port_id, attrs = constant_attributes.copy().popitem()
+    if ov_metatype in [
+        OVConvolutionMetatype,
+        OVDepthwiseConvolutionMetatype,
+        OVGroupConvolutionMetatype,
+        OVConvolutionBackpropDataMetatype,
+        OVGroupConvolutionBackpropDataMetatype,
+    ]:
```

Comment:
`OPERATIONS_WITH_WEIGHTS` already exist in OV backend, but it has LSTM, GRU and Embedding layers as well 
https://github.com/openvinotoolkit/nncf/blob/develop/nncf/openvino/graph/metatypes/groups.py#L165-L175

Reply:
It might be the marker that our graph builder is not applicable for all types from the `OPERATIONS_WITH_WEIGHTS` or `OPERATIONS_WITH_WEIGHTS` contains not correct meta types, isn't it?
At least, I'm not sure that it's good practice to hard-code such lists in the methods.

Reply:
This is how it's done in our QAT backends:
Torch: https://github.com/openvinotoolkit/nncf/blob/develop/nncf/torch/dynamic_graph/layer_attributes_handlers.py#L68
TF: https://github.com/openvinotoolkit/nncf/blob/develop/nncf/tensorflow/graph/converter.py#L745

We need a subset of metatypes that have the same type of layer attributes, and this subset is not always equal to `OPERATIONS_WITH_WEGIHTS` metatypes subset